### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by stephenkati**
### I created a form of UI for school library app.

- Created a `app.rb` file that serves as my console app entry-point. Has the following methods:
1. List all books.
2. List all people.
3. Create a person (teacher or student).
4. Create a book.
5. Create a rental.
6. List all rentals for a given person id.

- In `main.rb` I defined the entry point, this will be a method called main that is invoked at the end of your file. This method does the following:
1. Presents the user with a list of options to perform.
2. Lets users choose an option.
3. If needed, ask for parameters for the option.
4. Quits the app.

### CODE REVIEW

File: rental.rb [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
-  attr_accessor :date, :book, :person
+  attr_accessor :id, :date, :book, :person
 
   def initialize(date, book, person)
+    @id = Infor.generate_rental_id
     @date = date
     assign_book(book)
     assign_person(person)
+    Infor.rentals << self
+  end
```
Comment (elisha2kyakpo1) [inline rental.rb:10]: - [ ] I got an error when I tried to create a rental, kindly take a look at the image I have attached and make the necessary changes.

rental

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @stephenkati, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
✔️ The works nicely except for the rentals class functionaly👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: rental.rb (hunk 1 above)
Comment (stephenkati) [reply rental.rb:10]: Oh I realised my problem with selecting user by id when creating a rental, thanks for the feedback @elisha2kyakpo1

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @stephenkati     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**